docs/features/REST-sphubsitedata-type.md

```diff
@@ -0,0 +1,21 @@
+---
+title: SPHubSiteData REST type
+description: SPHubSiteData REST type
+ms.date: 2/26/2018
+---
+
+# SPHubSiteData REST type
+
+> [!IMPORTANT]
+> The hub sites feature is currently in preview and is subject to change. It is not currently supported for use in production environments.
+
+Contains data describing a SharePoint Hub site.
+
+|Name     |Type   |Description |
+|---------|-------|------------|
+|themeKey |string |  |
+|name     |string |The name of the hub site. |
+|url  |string   |Full URL of the hub site. |
+|LogoUrl |string |The URL of a logo to use in the hub navigation. |
+|usesMetadataNavigation |Boolean |  |
```

Comment:
Could say "True if publishing features are enabled on the hub parent site."

Reply:
And for the navigation property below this one: 
"Hub site navigation. If `usesMetadataNavigation` is true, this will be null. See below for navigation node properties."

I realized that `SPWeb.SerializableNavigationNode` is an undocumented internal type--it's similar to [.NET SiteMapNode](https://msdn.microsoft.com/en-us/library/system.web.sitemapnode%28v=vs.110%29.aspx) but only has a subset of properties and some naming differences. Possibly rather than using the type name, we should just call it a "list of navigation nodes". 

The properties could be documented in another table below the existing one. (I'd say put them in a bulleted list inside the table cell, but apparently it's hard to get lists and tables to play nice in Markdown.)

```
### Navigation node properties

|Name     |Type                     |Description                        |
|---------|-------------------------|-----------------------------------|
|Id       |number                   |ID of the navigation node          |
|Title    |string                   |Text of the navigation node        |
|Url      |string                   |URL the navigation node points to  |
|ParentId |number                   |ID of the parent navigation node   |
|Children |List of navigation nodes |List of child navigation nodes     |
```

(The class has a couple more properties which you can see in the REST responses, but those shouldn't be included here since they're not documented elsewhere and may change.)

Reply:
@ecraig12345  Thanks for this table! Added.